src/Status.h

```diff
@@ -31,6 +31,7 @@ namespace Status {
 /// would be present there can be a mixing of updated and non-updated
 /// information, which the mutex allows to avoid.
 class MessageInfo {
+  friend std::pair<double, double> messageSize(const MessageInfo &Information);
```

Comment:
This probably should just be a method in the class, there is no obvious need for it to be a friend.